### PR TITLE
Update calico to v3.18.1

### PR DESCRIPTION
**Pull request by surajssd**
- Add script to download and push images.

---

### Release Notes

Before updating control plane update the Calico CRDs by running:

```bash
kubectl apply -f ./lokomotive-0.x.x/assets/charts/control-plane/calico/crds/
lokoctl cluster apply -v --skip-components
```

### CODE REVIEW

**Review by meyskens — APPROVED** [review]
LGTM

File: assets/charts/control-plane/calico/scripts/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+```bash
+docker login -u <username> quay.io
+```
+
+### Execute script
+
+Run the following command in terminal:
+
+```bash
+./calico-images.sh v3.18.1
```
Comment (knrt10) [inline assets/charts/control-plane/calico/scripts/README.md:27]: I think we can use variable here for version. What do you think @surajssd?

**Review by knrt10 — COMMENTED** [review]
Small question

File: assets/charts/control-plane/calico/scripts/README.md (hunk 1 above)
Comment (surajssd) [reply assets/charts/control-plane/calico/scripts/README.md:27]: I don't understand what do you mean? It is an example. With new calico release pass `v3.19.x` or whatever.

Comment (knrt10) [reply assets/charts/control-plane/calico/scripts/README.md:27]: NVM, not so important

**Review by knrt10 — APPROVED** [review]
+1

**Pull request merged**